### ISSUE TITLE
on OpenBSD, Use of "seq" causes "not found" message when configure shows status

### ISSUE THREAD

**Issue by andy5995** [geany/geany#2054]
https://github.com/geany/geany/blob/c825b581bfabf6912eeb737a4af607e70c6c07a9/m4/geany-status.m4#L30

Due to `ksh` being the default shell on OpenBSD. But changing that line to something like

    for i in {1..$total};

may be a a good solution, if that's shell-portable.

```
config.status: executing libtool commands
config.status: executing default-1 commands
config.status: executing po/stamp-it commands
./configure[22990]: seq: not found

Install Geany in                 : /usr/local
Compiling Git revision           : cd267de2
Using GTK version                : 2.24.32
Enable binary relocation         : no
Build with plugin support        : yes
Use (UNIX domain) socket support : yes
Use virtual terminal support     : yes
Build HTML documentation         : yes
Build PDF documentation          : no
Build API documentation          : no
Generate GtkDoc header           : no

Configuration is done OK.
```

**Comment by elextr** [comment]
> may be a a good solution, if that's shell-portable.

sadly not

As @codebrainz said, its running the `seq` program, so its not shell related.

**Comment by elextr** [comment]
As I said, geany is in openbsd ports, so it builds there, so either seq is available or configure is being modified by the ports build?  I guess this is your chance to expand your BSD knowledge and learn how ports are built :grin:

Disclaimer: I am not a BSDist

**Comment by codebrainz** [comment]
> I installed the coreutils package, but seq still isn't available.

Do they call it `gseq` or something?

**Comment by elextr** [comment]
Yup, "All the binaries are prefixed by the letter g to differentiate them with
the standard applications with the same name." [from](http://ports.su/sysutils/coreutils)

**Comment by andy5995** [comment]
Yeah, gseq, which is apparently in the coreutils package.

> As I said, geany is in openbsd ports, so it builds there, so either seq is available or configure is being modified by the ports build? 

I don't see it's being modified. The maintainer's probably ignoring the output. All seq is used for is printing a bunch of hyphens. As shown in my output above, `configure` completes.

This issue is very trivial and only appearance-related.

**Comment by codebrainz** [comment]
> This issue is very trivial and only appearance-related.

Agree. Would it be enough to simply pipe stderr to `/dev/null` to hide the error on other systems?

**Comment by andy5995** [comment]
That would work, but imo adding a condition (and extra code) for this case may be too clunky.

Another option would be to remove the repeating hyphens as a separator (and therefore the use of `seq` and replace it with a line break and some text. Ex:


```
config.status: executing libtool commands
config.status: executing default-1 commands
config.status: executing po/stamp-it commands

    Build/configuration result:

Install Geany in                 : /usr/local
Compiling Git revision           : cd267de2
Using GTK version                : 2.24.32
Enable binary relocation         : no
Build with plugin support        : yes
Use (UNIX domain) socket support : yes
Use virtual terminal support     : yes
Build HTML documentation         : yes
Build PDF documentation          : no
Build API documentation          : no
Generate GtkDoc header           : no

Configuration is done OK.
```

**Comment by codebrainz** [comment]
I'm indifferent, any of the above is fine by me.

**Comment by elextr** [comment]
Yeah, since its just prettifying text its fine to remove.

**Comment by elextr** [comment]
@andy5995 Hate your last edit, Just make it plain text like it was.
